app/code/Magento/Catalog/Test/Mftf/Section/AdminCreateProductAttributeSection.xml

```diff
@@ -50,6 +50,11 @@
         <element name="StorefrontPropertiesSectionToggle" type="button" selector="#front_fieldset-wrapper"/>
         <element name="visibleOnCatalogPagesOnStorefront" type="select" selector="#is_visible_on_front"/>
     </section>
+    <section name="ManageLabelsSection">
```

Comment:
Could you please create a separate Section file, and include it in `ProductAttributePage` page?